modules/exploits/windows/local/capcom_sys_exec.rb

```diff
@@ -10,6 +10,7 @@ class MetasploitModule < Msf::Exploit::Local
   include Msf::Post::Windows::Priv
   include Msf::Post::Windows::Process
   include Msf::Post::Windows::ReflectiveDLLInjection
+  prepend Msf::Remote::Exploit::AutoCheck
```

Comment:
With this minor change the module was able to be loaded into the framework and was tested successfully.
This should also correct the CI failures. 
```suggestion
  prepend Msf::Exploit::Remote::AutoCheck
```